src/model/immutable/EditorState.js

```diff
@@ -210,6 +210,10 @@ class EditorState {
     return this.getImmutable().get('inlineStyleOverride');
   }
 
+  static setInlineStyleOverride(editorState: EditorState, inlineStyleOverride: DraftInlineStyle): EditorState {
+    return EditorState.set(editorState, { inlineStyleOverride });
+  }
```

Comment:
Oops, my bad. I saw that in the guidelines & then forgot about it. Have you considered adding it to the eslint config?

Updated now.